lib/dumbo/task/base.rb

```diff
@@ -31,8 +31,12 @@ def run!(overlord)
         @id = MultiJson.load(response.body)["task"]
       end
 
-      def finished?
-        uri = URI(@overlord + "/#{@id}/status")
+      def finished?(redirect)
```

Comment:
If its called without the redirect param somewhere (which is most likely the case) it has to be

```
      def finished?(redirect = nil)
```

Reply:
Done